catalog/app/containers/Bucket/PackageDialog/FilesInput.tsx

```diff
@@ -1565,6 +1601,10 @@ export function FilesInput({
           )}
         </HeaderTitle>
         <M.Box flexGrow={1} />
+        <M.Button size="small" onClick={onAddFolder}>
```

Comment:
imo an icon button would look nicer

Reply:
I have two options, both with icons:
1. button near "undo changes" looks nicer
2. button near "add local files" looks more relevant

![Screenshot from 2024-06-18 17-54-05](https://github.com/quiltdata/quilt/assets/533229/e546daad-9887-4fd1-828e-66cf7a53448c)
![Screenshot from 2024-06-18 17-54-18](https://github.com/quiltdata/quilt/assets/533229/6b8a6a0d-bb90-48b2-8eff-2c14b578c2f0)

Reply:
> button near "undo changes" looks nicer

i think this one is better, tho i'd probably kept it on the right (so that "undo changes" appears to the left instead of pushing it -- up to you tho)